src/asgen/utils.d

```diff
@@ -370,6 +374,10 @@ body
             downloader.dataTimeout = dur!"seconds" (30);
             downloader.onReceive = (data) => onReceiveCb (dest, data);
             downloader.perform();
+            if ("last-modified" in downloader.responseHeaders) {
```

Comment:
It is very fast and I assume the hash table is also pretty small, so this is indeed a micro optimization. Not sure what the compiler does here and how smart it is in optimizing - would be pretty cool if it did optimize that bit :-)

Anyway, this comment was from the "pedantic" and maybe-do-it class of comments ^^